source/connectors/kafka-sink-properties.txt

```diff
@@ -0,0 +1,249 @@
+       | **Accepted Values**: An empty string or a fully qualified Java class name
+
+   * - delete.on.null.values
+     - boolean
+     - | Whether or not the connector tries to delete documents based on key when value is null.
+       |
+       | **Default**: ``false``
+       | **Accepted Values**: ``true`` or ``false``
+
+   * - max.batch.size
+     - int
+     - | The maximum number of sink records to possibly batch together for processing.
+       |
+       | **Default**: ``0``
+       | **Accepted Values**: An integer
+
+   * - max.num.retries
+     - int
+     - | How often a retry should be done on write errors.
+       |
+       | **Default**: ``3``
+       | **Accepted Values**: An integer
+
+   * - retries.defer.timeout
+     - int
+     - | How long in ms a retry should get deferred.
+       |
+       | **Default**: ``5000``
+       | **Accepted Values**: An integer
+
+   * - change.data.capture.handler
+     - string
+     - | The class name of the CDC handler to use for processing.
+       |
+       | **Default**: ``""``
+       | **Accepted Values**: An empty string or a fully qualified Java class name
+
+   * - field.renamer.mapping
+     - string
+     - | An inline JSON array with objects describing field name mappings. For example:
+       | ``[ { "oldName":"key.fieldA", "newName":"field1" }, { "oldName":"value.xyz", "newName":"abc" } ]``
+       |
+       | **Default**: ``[]``
+       | **Accepted Values**: A valid JSON array
+
+   * - field.renamer.regexp
+     - string
+     - | An inline JSON array with objects describing regexp settings. For example:
+       | ``[ {"regexp":"^key\\\\..*my.*$", "pattern":"my", "replace":""}, {"regexp":"^value\\\\..*$", "pattern":"\\\\.", "replace":"_"} ]``
+       |
+       | **Default**: ``[]``
+       | **Accepted Values**: A valid JSON array
+
+   * - key.projection.list
+     - string
+     - | A comma-separated list of field names for key projection.
+       |
+       | **Default**: ``""``
+       | **Accepted Values**: A comma-separated list of field names
+
+   * - key.projection.type
+     - string
+     - | The type of key projection to use.
+       |
+       | **Default**: ``none``
+       | **Accepted Values**:  ``none``, ``blacklist``, or ``whitelist``
+
+   * - post.processor.chain
+     - list
+     - | A comma-separated list of post-processor classes to process the data before saving to MongoDB.
+       |
+       | **Default**: ``[ com.mongodb.kafka.connect.sink.processor.DocumentIdAdder ]``
+       | **Accepted Values**: A list of fully qualified Java class names
+
+   * - rate.limiting.every.n
+     - int
+     - | After how many processed batches the rate limit should trigger (NO rate limiting if n=0)
+       |
+       | **Default**: ``0``
+       | **Accepted Values**: An integer
+
+   * - rate.limiting.timeout
+     - int
+     - | How long in ms processing should wait before continuing processing.
```

Comment:
Not sure exactly what it means for processing to wait before processing

Reply:
Good catch. I missed this grammatical error (from pre-existing documentation).